herd/libdir/asl-pseudocode/implementations.asl

```diff
@@ -34,6 +34,10 @@ type S1PIRType of S2PIRType;
 type SCRType of SCR_Type;
 type SCTLRType of SCTLR_EL1_Type;
 
+// Our extractor for the system register types does not yet support those.
```

Comment:
The commit title ""Remove some useless implementations"  does not match the diffs that rather show code moves and comments erasure.

Reply:
Commit message is updated.